src/modules/task-runner.js

```diff
@@ -2,7 +2,16 @@
 
 const _ = require('lodash');
 const logger = require('winston');
-const sleep = require('sleep');
+
+const getOptionalSleep = () => {
+  try {
+    return require('sleep');
+  } catch (error) {
+    return null;
```

Comment:
Personally, rather than returning null here I would return an object that has a "usleep" function  that does nothing. That also removes the need for the null check later.